fenix.json.example

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Maybe this file's name should be `fenix.json` instead. But since it is the first config file in the repo, it could also be called `config.json.example`. Then there should be a a line in the readme telling people to copy this file to `config.json` (which should be added to `.gitignore`).